apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/CharSequenceAssertionSamples.kt

```diff
@@ -0,0 +1,239 @@
+//TODO remove file with 1.0.0
+@file:Suppress("DEPRECATION")
+
+package ch.tutteli.atrium.api.infix.en_GB.samples.deprecated
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.samples.fails
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class CharSequenceAssertionSamples {
+
+    @Test
+    fun containsBuilder() {
+        expect("ABC") contains o exactly 1 value "A"
+
+        expect("ABBC") contains o atLeast 2 value "B"
+
+        fails {
+            expect("AAAAAA") contains o atMost 3 value "A"
+        }
+    }
+
+    @Test
+    fun containsNotBuilder() {
+        expect("ABC") containsNot o value "X"
+
+        fails {
+            expect("ABC") containsNot o value "B"
+        }
+    }
+
+    @Test
+    fun contains() {
+        expect("ABC") contains "B"
+
+        expect("ABC123") contains values("AB", 'C', 12)
+
+        // holds because `contains` does not search for unique matches
+        // use `contains o exactly 2 value "A"` to check if subject contains two "A"s
+        expect("ABC") contains values("A", "A")
+
+        fails {
+            expect("ABC") contains "X"
+        }
+
+        fails { // because subject does not contain all values
+            expect("ABC") contains values("A", 99)
+        }
+    }
+
+    @Test
+    fun containsNot() {
+        expect("ABC") containsNot "X"
+
+        expect("ABC") containsNot values("X", 'Y', 1)
+
+        fails {
+            expect("ABC") containsNot "B"
+        }
+
+        fails {
+            expect("ABC") containsNot values("B", "X")
+        }
+    }
+
+    @Test
+    fun containsRegexStringSingle() {
+        expect("ABC") containsRegex "A(B)?"
+
+        fails {
+            expect("ABC") containsRegex "X"
+        }
+    }
+
+    @Test
+    fun containsRegexStringMultiple() {
+        // all regex patterns match
+        expect("ABC") contains regexPatterns("A(B)?", "(B)?C")
+
+        // holds because `containsRegex` does not search for unique matches
+        // use `contains exactly 2 regex "A(B)?"` to check if subject contains the regex two times
```

Comment:
```suggestion
        // holds because `contains regexPatterns(...)` does not search for unique matches
        // use `contains exactly 2 regex "A(B)?"` to check if subject contains the regex two times
```